jest.config-ts.js

```diff
@@ -0,0 +1,37 @@
+
+ module.exports = {
+  extensionsToTreatAsEsm: [
+    '.ts',
+    '.tsx',
+  ],
+  moduleFileExtensions: [
+    'js',
+    'jsx',
+    'ts',
+    'tsx',
+  ],
+  moduleNameMapper: {
+    '\\.scss$': 'identity-obj-proxy',
+    '\\.svg$': '<rootDir>/tests/jest/mocks/svgrMock.jsx',
+  },
+  preset: 'ts-jest',
+  setupFiles: [
+    '<rootDir>/tests/jest/setupJest.js',
+  ],
+  setupFilesAfterEnv: [
+  ],
+  testEnvironment: '@happy-dom/jest-environment',
```

Comment:
I know it is outside scope of this issue, but maybe we could add a comment here why we use happy-dom.

Reply:
We do not have comment neither here nor at `jest.config.js`. I can of course add it there, but while this is temporary file and original file does not have it, it does not make much sense to me.